### PR TITLE
Empty Rails Engine

### LINKED ISSUE #1: New Rails engine
Create a new Rails engine `--full --mountable`

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
New empty Rails Engine with the cruft removed. Some dep's added
  to the gemspec but not yet used (rspec, etc).

  Closes #1

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco please review.

**Comment by michaelachrisco** [comment]
Other than lack of rubocop, it looks good. Are we going to rubocop after the generation occurs?

File: fagus.gemspec [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+s.files = Dir["{app,config,db,lib}/**/*", "Rakefile", "README.md"]
+
+  s.add_dependency 'mysql2'
+  s.add_dependency 'rails', '~> 4.2.2'
+
+  s.add_development_dependency 'capybara'
+  s.add_development_dependency 'factory_girl_rails'
+  s.add_development_dependency 'faker'
+  s.add_development_dependency 'mysql2'
```
Comment (michaelachrisco) [inline fagus.gemspec]: I don't believe you need to need to add the development dependency mysql twice. I could be wrong?

Comment (josephbridgwaterrowe) [inline fagus.gemspec]: Good point, I'll remove the development dep.

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco, added rubocop, please review.

Thanks.

**Comment by michaelachrisco** [comment]
@josephbridgwaterrowe Rebase and I will merge.

**Pull request merged**